### ISSUE TITLE
Can't stroe to clipboard while embbed in Notion

### ISSUE THREAD

**Issue by leofcshen** [Marcello09/notion-lock#3]
Notion is embed as iframe. While click copy to clipboard gets error [The Clipboard API has been blocked because of a permissions policy applied to the current document.]

**Comment by Marcello09** [comment]
Yes, this is an issue with embedded iframes, javascript clipboard API is disabled by default. To solve this, the notion team must change their code
